pom.xml

```
@@ -7,7 +7,7 @@
 
     <properties>
         <version.slf4j>1.7.32</version.slf4j>
-        <version.log4j2>2.14.1</version.log4j2>
+        <version.log4j2>2.15.1</version.log4j2>
```

There's no 2.15.1 yet, I assume you meant 2.15.0 :)
```suggestion
        <version.log4j2>2.15.0</version.log4j2>
```